backend/doc/apache.md

```diff
@@ -30,23 +30,36 @@ Create a file named `/etc/apache2/sites-available/tracim.conf` containing:
             Dav On
```

Comment:
Comment may be nice here. https://httpd.apache.org/docs/2.4/en/mod/mod_dav.html. 
I do not understand why we do need mod dav and what it is doing.

Anyway if this is useful only for webdav, it may make sense to put this only in webdav location.

Reply:
Apache.md documentation is come from tracim v1 I think. so some information need to be updated.

Reply:
I removed this line and didn't see any effect on webdav operations (tried creating/moving/deleting files, folders and workspaces).
As gives unauthenticated access to the frontend's code (luckily read-only as the directory/files are RO for the apache's user) I would recommend to remove it.